scanner/image/db/Dockerfile

```diff
@@ -16,34 +22,29 @@ ENV PATH="$PATH:/usr/pgsql-$PG_MAJOR/bin/" \
     PGDATA="/var/lib/postgresql/data/pgdata"
 ENV LANG=en_US.utf8
 
+COPY signatures/RPM-GPG-KEY-PGDG-13 /
 COPY scripts/docker-entrypoint.sh /usr/local/bin/
+COPY --from=postgres_rpms /rpms/postgres.rpm /rpms/postgres-libs.rpm /rpms/postgres-server.rpm /rpms/postgres-contrib.rpm /tmp/
 
 ARG POSTGRESQL_ARCH=x86_64
```

Comment:
This is now encapsulated as a part of the download.sh so let's remove all usages of this